.travis.yml

```diff
@@ -6,10 +6,16 @@ julia:
   - nightly
 notifications:
   email: false
+matrix:
+  allow_failures:
+  - julia: nightly
 script:
   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
   - julia -e 'Pkg.clone(pwd()); Pkg.build("Gadfly");'
   - julia -e 'Pkg.checkout("Compose")'
+  # because of a bug in Julia 0.6.2 we temporarily use --compilecache=no.
+  # The bug is fixed on master and will be included in 0.6.3 so when it
+  # has been releases, we can bump the requirement and remove the flag.
```

Comment:
this comment no longer needs to be here.  otherwise looks good.  all the regression tests pass?